database/mysql-ob/requirements.yaml

```diff
@@ -12,16 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: v1
-kind: Service
-metadata:
-  name: {{ template "ob-pattern-1.resource.prefix" . }}-ob-km-service
-  namespace : {{ .Release.Namespace }}
-spec:
-  selector:
-    deployment: wso2ob-km
-  sessionAffinity: ClientIP
-  ports:
-  - name: servlet-https
-    port: 9446
-    protocol: TCP
+dependencies:
+  - name: mysql-ob-berlin
+    version: "2.0.0-1"
+    repository: "file://../mysql-ob-berlin"
```

Comment:
@nalakagunathilaka let's not commit the file path based repository value. Let's set the expected, final Helm chart name in the official WSO2 Helm chart repository for each sub chart in this case.

Also, let's point to the official WSO2 Helm chart repository.

Once, the charts have been officially uploaded, please perform a test to ensure whether the added dependencies are properly downloaded when building this chart.

Reply:
It was decided to keep local repository paths like that. @Aaquiff please comment on this.

Reply:
@chirangaalwis since these charts are not used standalone, do we need to publish this as a separate chart?

Reply:
Yeah. +1, let's go with the file path in this chart.